pandas/tests/strings/test_case_justify.py

```diff
@@ -278,6 +278,8 @@ def test_center_ljust_rjust_mixed_object():
 
 
 def test_center_ljust_rjust_fillchar(any_string_dtype):
+    if any_string_dtype == "string[pyarrow_numpy]":
+        pytest.skip("Arrow logic is different")
     s = Series(["a", "bb", "cccc", "ddddd", "eeeeee"], dtype=any_string_dtype)
```

Comment:
So in this case the `"string[pyarrow]"` version (ArrowStringArray) was still falling back to the python object-dtype implementation? (since we are not skipping that one)

This might be a case where we could add an option to the pyarrow compute kernel that determines if it's aligned left or right for uneven centering?

Reply:
That would be the cleanest solution, yes "string[pyarrow]" fell back to python objects

Reply:
Not for this PR, but I think we should consider for the default string dtype for 3.0 to preserve the current behaviour (and so if necessary for now fallback to the python objects), _if_ our preferred option would be to keep this behaviour long term through a keyword in pyarrow. 
(otherwise that would be a change in behaviour again later, when pyarrow would support that)

Reply:
Regarding the above `center` behaviour, opened https://github.com/pandas-dev/pandas/issues/54807 to discuss changing this to fall back for now (to preserve the behaviour)